cmd/sourced/main.go

```diff
@@ -12,5 +15,25 @@ func main() {
 	composefile.SetVersion(version)
 	cmd.SetVersion(version)
 
+	checkUpdates()
+
 	cmd.Execute()
 }
+
+func checkUpdates() {
+	if version == "master" {
+		return
+	}
+
+	update, latest, err := release.FindUpdates(version)
+	if err != nil {
+		return
```

Comment:
Why not logging the error? It could be useful for the user to know that the version could not be check because of reasons.

Reply:
My view is that as a user, if I do a `sourced status` when I don't have connectivity, or GitHub is down, I don't want the command to show me errors for functionality I'm not interested in.

If we had a log file, I would log this silently.